aten/src/ATen/native/native_functions.yaml

```diff
@@ -1809,6 +1809,27 @@
     CPU: _s_where_cpu
     CUDA: _s_where_cuda
 
+- func: norm_except_dim(Tensor v, int64_t pow=2, int64_t dim=0) -> Tensor
+  variants: function
+
+# VariableType::_weight_norm does not want to be given a gap in the autograd graph,
+# so we don't define "dispatch" variants for it.
+- func: _weight_norm(Tensor v, Tensor g, int64_t dim=0) -> Tensor
+  variants: function
+
+- func: weight_norm_cuda_interface(Tensor v, Tensor g, int64_t dim=0) -> (Tensor, Tensor)
```

Comment:
I added the underscore decorations, but the latest push doesn't appear to be retriggering the CI (possibly because the PR is already approved).  For what it's worth I did build and run test_autograd.py one last time before pushing.

Thanks again for all your help!